### ISSUE TITLE
How to check gitleaks version on macos(fetched with Homebrew)

### ISSUE THREAD

**Issue by acearth** [zricethezav/gitleaks#434]
**Describe the bug**
By using `gitleaks --help` command, I found we can use `gitleaks --version` to check its version.
Since the gitleaks is fetched by homebrew, the output is 
```
$ gitleaks --version
Gitleaks uses LDFLAGS to pull most recent version. Build with 'make build' for version
```

Though the version can be fetched by `brew info gitleaks`, it's can be better if the result can be directly shown by the `gitleaks --version`.

**To Reproduce**

**Expected behavior**
Show gitleaks version which is widely supported.
**Screenshots**

**Basic Info (please complete the following information):**
 - OS: macos 10.15
 - Gitleaks Version: 6.0.0

**Additional context**

cc @zricethezav

**Comment by zricethezav** [comment]
Good call, looks like the [brew formula is wrong](https://github.com/Homebrew/homebrew-core/blob/master/Formula/gitleaks.rb#L17-L20). I believe this was introduced in v6+.

```
  def install
    system "go", "build", "-ldflags", "-X github.com/zricethezav/gitleaks/version.Version=#{version}",
                 *std_go_args
  end
```
Should be 
```
  def install
    system "go", "build", "-ldflags", "-X github.com/zricethezav/gitleaks/v6/version.Version=#{version}",
                 *std_go_args
  end
```

It should match the ldflags used in my build script: https://github.com/zricethezav/gitleaks/blob/master/Makefile#L5

**Comment by zricethezav** [comment]
fixed in 6.1.1.

`brew upgrade gitleaks`
